experimental/c-cpp/build_generator.py

```diff
@@ -52,10 +53,10 @@ def match_files(self, file_list):
 
   def is_matched(self):
     """Returns True if the build heuristic found matching files."""
-    for matches in self.matches_found:
-      if len(matches) == 0:
-        return False
-    return True
+    for found_matches in self.matches_found.values():
+      if len(found_matches) > 1:
```

Comment:
Why > 1 and not > 0? Can you add a comment here?

Reply:
It should be > 0 -- thanks for catching!